timely_beliefs/tests/test_belief_utils.py

```diff
@@ -31,3 +33,55 @@ def test_propagate_multi_sourced_deterministic_beliefs():
             == pd.Timestamp("2000-01-01 01:00:00+00:00")
         ].droplevel("belief_time"),
     )
+
+
+@pytest.mark.parametrize(
+    ("start", "periods", "resolution", "exp_event_values"),
+    [
+        (
+            "2022-03-27 01:00+01",
+            7,
+            "PT2H",
+            [2, 3, 5, 7],
+        ),  # DST transition from +01 to +02 (spring forward, contracted event)
+        (
+            "2022-10-30 01:00+02",
+            7,
+            "PT2H",
+            [2, 5, 7],
```

Comment:
I there no 3 in this one due to drop_duplicates?

Reply:
No. Actually, you prompted me to discover that the call to `drop_duplicates` is no longer needed since I started using the `join` operation in the line following it.

Events on either side of the DST transition are 2 hours apart, and fall on even hours on either side, so midnight, 2 AM, 4 AM etc. Here there is no 3 because the event starting at 2 AM (local time, +02) is 3 hours apart from the event starting at 4 AM (local time, +01).